Documentation/Signoffs/Raymond Mule-Signoff-Wireless Communication Subsystem(Updated).md

```diff
@@ -0,0 +1,37 @@
+# Wireless Communication Subsystem
+
+## Functionality- 
+The function of the wireless communication subsystem is to wirelessly transport data from the computing system to the user interface subsystem quickly and over a long distance. 
+
+## Constraints- 
+| Constraint | Source |
+|------------|--------|
+| 1. The system shall be able to communicate with the computing subsystem from a minimum distance of 700-meters| Analyzed from DARPA's constraints |
```

Comment:
Give more detail below the constraints section here about how you derived this. Don't give the explanation in the analysis section.